test/src/704-world_readable/setup_teardown

```diff
@@ -0,0 +1,80 @@
+create_world_readable_repo() {
+  echo ""
+  echo "*** CREATE a fresh repository named $CVMFS_TEST_REPO with user $CVMFS_TEST_USER"
+  create_empty_repo $CVMFS_TEST_REPO $CVMFS_TEST_USER || return $?
+
+  local short_name="$1"
+  local long_name="$2"
+  local symlink_name="$3"  
+
+  start_transaction $CVMFS_TEST_REPO || return $?
+
+  touch /cvmfs/$CVMFS_TEST_REPO/test1.txt
+  mkdir /cvmfs/$CVMFS_TEST_REPO/testdir
+  touch /cvmfs/$CVMFS_TEST_REPO/testdir/test2.txt
+
+  chmod 0400 /cvmfs/$CVMFS_TEST_REPO/test1.txt
+  chmod 0400 /cvmfs/$CVMFS_TEST_REPO/testdir/test2.txt
+  chmod 0500 /cvmfs/$CVMFS_TEST_REPO/testdir
+
+  publish_repo $CVMFS_TEST_REPO || return 200
+  echo "*** FINISHED creating a fresh repository named $CVMFS_TEST_REPO with user $CVMFS_TEST_USER"
+  echo ""
+}
+
+# add_some_tmp_file_to_repo() {
+#   echo "Adding some random file to the repo $CVMFS_TEST_REPO"
+#   start_transaction $CVMFS_TEST_REPO || return $?
+#   local tmpfile="$(mktemp /cvmfs/$CVMFS_TEST_REPO/tmpfile.XXXXX)"
+#   publish_repo $CVMFS_TEST_REPO || return 200
+# }
+
+private_mount_world_readable() {
```

Comment:
This function as well as the `private_mount_not_world_readable` don't seem to add much. Consider using `do_local_mount_as_root` directly or at least merging the tow functions into one.